src/plan_expand_hypertable.c

```diff
@@ -182,7 +202,15 @@ ts_plan_expand_hypertable_chunks(Hypertable *ht,
 	 */
 	hri = ts_hypertable_restrict_info_create(rel, ht);
 	ts_hypertable_restrict_info_add(hri, root, restrictinfo);
-	inh_oids = find_children_oids(hri, ht, AccessShareLock);
+
+	if (should_order_append(root, rel, ht, &reverse))
+	{
+		((TimescaleDBPrivate *) rel->fdw_private)->appends_ordered = true;
```

Comment:
should we check that rel->fdw_private != NULL?

Reply:
done